### PR TITLE
Localize collaboration settings.

### LINKED ISSUE #2764: Translate dialogs without translation
- Collaborators
image

- New page's editor of Wiki
image

- Gravatar email dialog in profile.tmpl (it's a concatenation string)
image

Thank you and saludos! :+1:

### PR DESCRIPTION
**Pull request by andreynering**
Closes #2764

### CODE REVIEW

File: models/repo_collaboration.go [hunk 1]
```diff
@@ -16,16 +16,17 @@ type Collaboration struct {
...
-		return "Read"
+		return "repo.settings.collaboration.read"
 	case ACCESS_MODE_WRITE:
-		return "Write"
+		return "repo.settings.collaboration.write"
 	case ACCESS_MODE_ADMIN:
-		return "Admin"
+		return "repo.settings.collaboration.admin"
+	default:
+		return "repo.settings.collaboration.undefined"
```
Comment (bkcsoft) [inline models/repo_collaboration.go]: Any reason for adding a default?

**Comment by bkcsoft** [comment]
Commented, otherwise it looks good to me :+1:

File: models/repo_collaboration.go (hunk 1 above)
Comment (andreynering) [inline models/repo_collaboration.go]: This was on the bottom of the function before. I find it more Go idiomatic to put this inside the switch block, but the result is the same.

Comment (bkcsoft) [inline models/repo_collaboration.go]: Missed that :smile:

**Comment by Unknwon** [comment]
Thanks!

**Pull request merged**